spec/controllers/sorry_controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+describe SorryController, type: :controller do
+  describe 'get #unable' do
+    it 'returns http success' do
+      get :unable
+      expect(response).to have_http_status(:success)
```

Comment:
:+1: on this test.

It seems strange that this page would return a successful status code though.  We should probably respond with an appropriate http status code for an error state.